packages/angular/projects/clr-angular/src/timeline/enums/timeline-step-state.enum.ts

```diff
@@ -6,8 +6,8 @@
 
 export enum ClrTimelineStepState {
   NOT_STARTED = 'not-started',
-  CURRENT = 'current',
+  CURRENT = 'info',
   PROCESSING = 'processing',
```

Comment:
I am not sure what color the `processing` has to be, maybe is `info` but I need advice here.

Reply:
The current step is blue. Info is blue. But these colors are driven by the CSS. So I would think you would want to update those selectors? Or the styles? I don't think you need to update the enum. That's not going to solve it.

Reply:
https://github.com/bbogdanov/clarity/blob/migrate-to-core-icons/packages/angular/projects/clr-angular/src/timeline/timeline-step.ts#L21

https://github.com/bbogdanov/clarity/blob/migrate-to-core-icons/packages/angular/projects/clr-angular/src/timeline/timeline-step.ts#L40

I believe two will lines will give some clarity on why I change it like that. There can be a better way but I might introduce a breaking change. What do you think?

Reply:
Processing is a `clr-spinner` component when the timeline step state is `procesing`. I would expect that the color will be updated there when I get tothat part ofthe review :) 

I agree with Scott. I don't think these values need to change. If they do, we should update the Enum constant value.

Reply:
[Commit](https://github.com/vmware/clarity/pull/5190/commits/a32306bf19bc025b2a3424567f35865ed8702860) I believe this solution we will agree is much better than the previous one